### PR TITLE
PEP8ize II (1) correct indentation

### PR DESCRIPTION
**Pull request by rob-baron**
PEP8 errors: E101, E112, E113, E114

After finding several cases where agressively flake8ing will cause Dijango to not
function correctly, and figuring out why the McCabe Cyclometric Complexity measure
is not truly meaningful, have decided to take the suggestion of a number of people
who suggested that we gradually increase the number of errors that we handle.

I have started with these four as they all deal with incorrect indentation which is
a problme for any python code.

### CODE REVIEW

**Review by kamfonik — APPROVED** [review]

**Comment by kamfonik** [comment]
Looks good to me, I think doing the errors in small batches is a good plan.

**Comment by xuhang57** [comment]
We need a plan for how/when to do this. Since we basically ignoring everything:

`+ignore = F401,F402,F403,F404,F405,F406,F407,F601,F602,F621,F622,F631,F701,F702,F703,F704,F705,F706,F707,F811,F812,F821,F822,F823,F831,F841,E111,E121,E122,E123,E125,E126,E127,E128,E129,E131,E201,E202,E203,E211,E223,E224,E225,E226,E228,E266,E231,E241,E242,E251,E261,E262,E265,E271,E272,E301,E302,E303,E304,E306,E305,E401,E402,E501,E502,E503,E701,E702,E703,E711,E712,E713,E731,W191,W291,W292,W293,W391,W503,W601,W602,W603,W604,W402,W403,W404,W405,W801,W802,W803,W804,W805,W806,W806,W901`

Reference: 
http://pep8.readthedocs.io/en/release-1.7.x/intro.html#error-codes

**Review by xuhang57 — CHANGES_REQUESTED** [review]
We should start fixing the code base

**Comment by kamfonik** [comment]
@xuhang57 one plan could be to have a single PR that is open until this is done.  Each new commit addresses a specific error (or a group of related errors), and removes the error from the ignore list.    Travis will run when it is pushed and double check that you got it right.  

Repeat until the ignore list is empty (or contains only errors we actually do want to ignore), then squash & merge.

**Comment by xuhang57** [comment]
@kamfonik  Great plan! Thanks Laura. Yes, I notice that travis set this PR "all checks have passed" since lots of the error codes have ignored. 

I could have a PR for enforcing the rest of PEP8 rules.

Thanks

**Review by xuhang57 — APPROVED** [review]
I will clean up whatever left

**Comment by rob-baron** [comment]
Lucas,

Yes, Laura just iterated my basic game plan.  There are a number of errors that are format related and will be checked in soon.  However, as pointed out by Kristi and Jermey we shouldn't be too aggressive with all of the errors.  For example, imports are sometimes done for their side effects and will appear unused (and this seems to be the case in atmosphere)..

Additionally, there are some things that I have found django doing that are not quite PEP8 standard (having import files in odd locations in a file), that I will need to suppress.

Basically, it is the low hanging fruit first, and then the more complex things later.

Also, we will suppress the cyclomatic complexity as it is not considered a useful measure of complexity.

**Comment by xuhang57** [comment]
Based on: https://docs.djangoproject.com/en/1.9/internals/contributing/writing-code/coding-style/

Just a few exceptions we that Django is doing differently from pep8, which does bother us too much at all. 

We basically cannot ignore all of this since in each new PR, potential errors/bugs would be introduced without having a valid CI. Also, we haven't written any new unit test/mock test/ and functional test for each PR. 

I don't think we should merge anything to the code base significantly without having these all done.
